uiowa/uiowa
# Implement hook_cron in uiowa_core for hourly tag invalidation.

Resolves #4842

# How to test

The default dev site sitenow_events.settings have been changed to use the content.dev.drupal.uiowa.edu API. Check with `drush @default.dev cget sitenow_events.settings`.

- Open https://default.dev.drupal.uiowa.edu/about in an anonymous session to cache a response in Varnish.
- Edit https://content.dev.drupal.uiowa.edu/test-4 like the title or something.
- Wait an hour for cron to run. Cron is running on the hour for uiowa.dev
- See if the https://default.dev.drupal.uiowa.edu/about page refreshes.
